node/src/service.rs

```diff
@@ -109,22 +117,25 @@ impl NativeExecutionDispatch for DolphinRuntimeExecutor {
     }
 }
 
+/// We use wasm executor only now.
+pub type DefaultExecutorType = WasmExecutor<HostFunctions>;
+
 /// Full Client Implementation Type
-pub type Client<R, E> = TFullClient<Block, R, NativeElseWasmExecutor<E>>;
+pub type Client<RtApi> = TFullClient<Block, RtApi, DefaultExecutorType>;
```

Comment:
Can you name it as `RuntimeApi`? It should be more readable.

Reply:
done.